pkg/controller/storagecluster/noobaa_system_reconciler.go

```diff
@@ -1,32 +1,38 @@
-package storageclusterinitialization
+package storagecluster
 
 import (
 	"context"
-	"fmt"
-	"os"
 	"reflect"
 
 	"github.com/go-logr/logr"
 	nbv1 "github.com/noobaa/noobaa-operator/v2/pkg/apis/noobaa/v1alpha1"
 	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
 	"github.com/openshift/ocs-operator/pkg/controller/defaults"
+	scinit "github.com/openshift/ocs-operator/pkg/controller/storageclusterinitialization"
+	statusutil "github.com/openshift/ocs-operator/pkg/controller/util"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
-func (r *ReconcileStorageClusterInitialization) ensureNoobaaSystem(initialData *ocsv1.StorageClusterInitialization, reqLogger logr.Logger) error {
+func (r *ReconcileStorageCluster) ensureNoobaaSystem(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+
+	initialData := &ocsv1.StorageClusterInitialization{}
```

Comment:
Why are we still fetching the StorageClusterInit? We should be able to get all the information we need from the StorageCluster, and set that as the owning controller.

Reply:
you're right! we don't need the initialization object at all here anymore, I'll fix this

Reply:
fixed, good catch :+1: